test/k8sT/Nightly.go

```diff
@@ -357,6 +358,20 @@ var _ = Describe("NightlyExamples", func() {
 	})
 
 	Context("Cilium DaemonSet from example", func() {
+		// validatedImage validates that the cilium image is the same as the
+		// given image
+		validatedImage := func(image string) {
+			By(fmt.Sprintf("Checking that installed image is %q", image))
+
+			filter := `{.items[*].status.containerStatuses[0].image}`
+			data, err := kubectl.GetPods(
```

Comment:
Not blocking, but we should change the existing `GetCiliumPods` to be named `GetCiliumPodsNameList` and then add `GetCiliumPods` which is a wrapper around `kubectl.GetPods(helpers.KubeSystemNamespace, "-l k8s-app=cilium")`. I can file this as a separate ticket if you like? Let me know.

Reply:
I think that we should discuss this :-( For example @aanm disagrees to have all in helpers and is the reason that he added API support. 

I do not see a real value to have a different way to get pods in Cilium. But again, we should discuss in a conf call or something and set the basics of the test.